midterm-2023-11-06/code/5163-mihaela-milosevska/script.ts

```diff
@@ -0,0 +1,82 @@
+interface album{
+    name: string;
+    year; number;
+}
+
+interface band {

```

Comment:
Conventionally, interfaces (and classes) in TypeScript are written with a Uppercase first letter, so `Band` instead of `band`. The code after actually uses `Band` :D +5